### PR TITLE
[ISSUE-18] Add parcel bundling for modern syntax and better modularization

### PR DESCRIPTION
**Pull request by mfmendiola**
**WHAT**
Add parcel bundling for modern syntax and better modularization

**WHY**
- Cleaner modularization, variable definitions
- latest syntax via babel

**ISSUE**
https://github.com/michaelwschultz/space-shooter-led/issues/18

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,20 +4,26 @@
...
+    "@babel/preset-env": "^7.4.5",
+    "babel-jest": "^24.8.0",
     "faker-js": "^1.0.0",
     "jest": "^24.7.1"
   },
   "scripts": {
-    "start": "nodemon ./js/server.js",
-    "test": "jest --config ./config/jest.config.js"
+    "start": "nodemon --watch ./js/server.js ./js/server.js ",
+    "test": "jest --config ./jest.config.js"
```
Comment (michaelwschultz) [inline package.json]: Did you actually move this file? Test is failing because it can't find it.

**Comment by mfmendiola** [comment]
Is there a way for you to  add me as an admin on this repo or  add  me to the  Now account? Can't  seem to  access it to troubleshoot.

**Comment by michaelwschultz** [comment]
@mfmendiola think you just need to run `yarn test` locally. The other thing that would be helpful is to have you as an admin on Circle but I don't think you can do that without a paid plan. Same with Now.

**Comment by mfmendiola** [comment]
Tests should be passing now, but `Now` is still broken. Think it has to do with the fact that we're running a separate server for parcel now

**Comment by michaelwschultz** [comment]
Not able to run this on a new machine.

**Pull request merged**